app/controllers/series_controller.rb

```diff
@@ -107,20 +107,20 @@ def create
   # PUT /series/1
   # PUT /series/1.xml
   def update
-    unless params[:series][:author_attributes][:ids]
+    unless series_params[:author_attributes][:ids]
       flash[:error] = ts("Sorry, you cannot remove yourself entirely as an author of a series right now.")
       redirect_to edit_series_path(@series) and return
     end
-    
-    if params[:pseud] && params[:pseud][:byline] && params[:pseud][:byline] != "" && params[:series][:author_attributes]
+
+    if params[:pseud] && params[:pseud][:byline] && params[:pseud][:byline] != "" && series_params[:author_attributes]
       valid_pseuds = Pseud.parse_bylines(params[:pseud][:byline])[:pseuds] # an array
       valid_pseuds.each do |valid_pseud|
-        params[:series][:author_attributes][:ids] << valid_pseud.id rescue nil
+        series_params[:author_attributes][:ids] << valid_pseud.id rescue nil
```

Comment:
I have had a quick play with this and have managed to convinced myself there is an issue.

series_params is a method so adding things on to the return value doesn't feel like it should work.

Just to be sure I cheated horribly and clone the repository and made this bit.

        puts
        puts series_params[:author_attributes][:ids]
        puts valid_pseud.id
        series_params[:author_attributes][:ids] << valid_pseud.id rescue nil
        puts series_params[:author_attributes][:ids]

Then ran the tests and I get.

.....*
5
7
5
...

Could you check to see if I am being stupid ?